index.d.ts

```diff
@@ -28,6 +28,25 @@ declare namespace cliTruncate {
 		```
 		*/
 		readonly space?: boolean;
+
+		/**
+		Truncate at different positions.
```

Comment:
This needs the same description as the readme.

Reply:
Done